bin/varnishd/cache/cache_conn_pool.c

```diff
@@ -222,6 +224,8 @@ VCP_AddRef(struct conn_pool *cp)
  * Release Conn pool, destroy if last reference.
  */
 
+static struct vrb dying_cps = VRBT_INITIALIZER(&dying_cps);
```

Comment:
Please declare it just below `conn_pools`.

For this, you should declare the tree head separately:

```diff
-static VRBT_HEAD(vrb, conn_pool) conn_pools = VRBT_INITIALIZER(&conn_pools);
+VRBT_HEAD(vrb, conn_pool);
 VRBT_GENERATE_REMOVE_COLOR(vrb, conn_pool, entry, static)
@@ ... all the VRBT_GENERATE calls ... @@
 VRBT_GENERATE_INSERT(vrb, conn_pool, entry, vcp_cmp, static)
+VRBT_GENERATE_NEXT(vrb, conn_pool, entry, static);
+VRBT_GENERATE_MINMAX(vrb, conn_pool, entry, static)
+
+static struct vrb conn_pools = VRBT_INITIALIZER(&conn_pools);
+static struct vrb dead_pools = VRBT_INITIALIZER(&dead_pools);
```

Note that I changed the head name for consistency.

Maybe as a followup commit we could rename `struct vrb` to `struct vrb_vcp`.